macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -360,7 +360,7 @@ open class AvatarView: NSView {
 		var updatedBackgroundColor = avatarBackgroundColor
 		var updatedInitialsColor = initialsFontColor
 		if !isCustomAvatarBackgroundColorConfigured || !isCustomAvatarInitialColorConfigured {
-			let defaultColorSet = AvatarView.getInitialsColorSet(fromPrimaryText: contactEmail, secondaryText: contactName)
+			let defaultColorSet = AvatarView.getInitialsColorSet(fromPrimaryText: contactName, secondaryText: contactEmail)
```

Comment:
If order matters, is it worth it to update getInitialColorSet's param names to reflect that? If we can't due to breaking public change, should we ensure it's at least documented well? Is there any other input verification we can do on secondaryText to ensure we have these ordered correctly?

Reply:
The params portion is a little complicated right now because our params don"t even align with web. Documentation might be the better option as of now until that gets resolved.

Reply:
Commented!